catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/NitfHeaderAttribute.java

```diff
@@ -240,12 +240,6 @@
             NitfHeader::getComplexityLevel,
             BasicTypes.INTEGER_TYPE);
 
-    public static final NitfHeaderAttribute FILE_DATE_AND_TIME_ATTRIBUTE  = new NitfHeaderAttribute(
-            FILE_DATE_AND_TIME,
-            "FDT",
-            header -> convertNitfDate(header.getFileDateTime()),
-            BasicTypes.DATE_TYPE);
```

Comment:
This attribute was removed because the file date and time is already being stored under `FILE_DATE_AND_TIME` by the `FILE_DATE_AND_TIME_CREATED_ATTRIBUTE`.